### PR TITLE
Add python interface for the Gitter api

### PR DESCRIPTION
**Pull request by MichaelYusko**
Add python interface for Gitter API, and in my free time I will make improvements to [GitterPy](https://github.com/MichaelYusko/GitterPy)

### CODE REVIEW

File: 09.Third-party-API-wrappers.md [hunk 1]
```diff
@@ -18,3 +18,7 @@ You can also access the Gitter API through one of the 3rd party libraries that m
 ## PHP
 
 * [gitter-api](https://github.com/SerafimArts/gitter-api) - Gitter API implementation for php 7.0+ allowing sync, async and streaming access.
+
+## Python
+
+* [gitterpy](https://github.com/MichaelYusko/GitterPy) - Python Interface for the Gitter API
```
Comment (MadLittleMods) [inline 09.Third-party-API-wrappers.md]: small-nit, `interface` shouldn't be capitalized

**Comment by MadLittleMods** [comment]
Thanks @MichaelYusko, only a small nit to fix up.

**Comment by MichaelYusko** [comment]
@MadLittleMods Sorry, it's my mistake, thanks! fixed 😄

**Comment by MadLittleMods** [comment]
♥️ @MichaelYusko It will appear when the developer site gets deployed again.

**Comment by MichaelYusko** [comment]
@MadLittleMods it's really cool, I'm waiting;) I need celebrate this event, so I go buy beer:)

> in my free time I will make improvements to GitterPy
😺

**Pull request merged**